install.sh

```diff
@@ -10,7 +10,7 @@ docker exec shopware bash -c "bin/console sales-channel:update:domain ${APP_URL}
 docker exec shopware bash -c "sudo mysql -u root -proot shopware -e \"SELECT @RULE_ID := id FROM rule WHERE name = 'All customers'; UPDATE shipping_method SET availability_rule_id = @RULE_ID;\""
 
 # Install Adyen Plugin 
-docker exec shopware bash -c 'composer require adyen/adyen-shopware6'
+docker exec shopware bash -c "composer require adyen/adyen-shopware6:'${SHOPWARE_PLUGIN_VERSION}'"
```

Comment:
It would be nice to assign a default value to this if it's not specified. The install.sh will break when this value isn't provided (which will break gitpod / or if you run it locally*) 

[Optional] Alternative solution:

if `${SHOPWARE_PLUGIN_VERSION}` is not set, it will run a specific (latest) version, 
Otherwise it will override the :`${SHOPWARE_PLUGIN_VERSION}` value